test/Microsoft.AspNetCore.AzureAppServices.FunctionalTests/AzureFixture.cs

```diff
@@ -98,8 +98,51 @@ public string GetTimestampedName(string name)
             return name + Timestamp;
         }
 
+        private Dictionary<string, List<Task<IWebApp>>> _deploymentCache = new Dictionary<string, List<Task<IWebApp>>>();
+        private int _predeploymentId;
+
         public async Task<IWebApp> Deploy(string template, IDictionary<string, string> additionalArguments = null, [CallerMemberName] string baseName = null)
         {
+            List<Task<IWebApp>> deployments;
+
+            void DeployBatch()
+            {
+                var maxId = _predeploymentId + 5;
```

Comment:
At the end of our tests we'll have a few unused Web Apps?

Reply:
Yep, I don't think it's too bad but we can add environment variable to limit total number on CI